Makefile

```diff
@@ -30,3 +30,19 @@ clean:
 	rm -rf $(PYTHON_ENV_ROOT)
 
 envs: env packaging-env
+
+
+# testing #####################################################################
+$(PYTHON_TESTING_ENV)/.created: REQUIREMENTS.qa.txt
+	rm -rf $(PYTHON_TESTING_DEV) && \
```

Comment:
What's supposed to be deleted here? When I ran `make qa` it seems that `PYTHON_TESTING_DEV` was empty.

Reply:
You are right. (This typo must be in different projects by now, as the initial commit goes back to 2021...)